### PR TITLE
Cleanup rw old secret

### PR DESCRIPTION
**Pull request by mcbenjemaa**
towards https://github.com/giantswarm/giantswarm/issues/24222

The old secret `remotewrite`  was not cleaned up when we implemented the `RemoteWrite` CR.

## Checklist

I have:

- [X] Described why this change is being introduced
- [ ] Separated out refactoring/reformatting in a dedicated PR
- [X] Updated changelog in `CHANGELOG.md`

### CODE REVIEW

**Review by marieroque — APPROVED** [review]

**Review by hervenicol — APPROVED** [review]
Can't say much about the code, but I always love these kind of stats :+1: 
image

Good for me if tested.

File: service/controller/resource/monitoring/remotewriteconfig/resource.go [hunk 1]
```diff
@@ -1,86 +0,0 @@
...
-}
-
-func getObjectMeta(ctx context.Context, v interface{}) (metav1.ObjectMeta, error) {
-	cluster, err := key.ToCluster(v)
-	if err != nil {
-		return metav1.ObjectMeta{}, microerror.Mask(err)
-	}
-
-	return metav1.ObjectMeta{
-		Name:      key.RemoteWriteSecretName(),
```
Comment (mcbenjemaa) [inline service/controller/resource/monitoring/remotewriteconfig/resource.go:59]: Remove this from Key

**Comment by mcbenjemaa** [comment]
@TheoBrigitte This PR is Ready.
The previous patch release was Done

**Comment by QuentinBisson** [comment]
Need to fix the changelog

**Review by QuentinBisson — APPROVED** [review]

**Review by TheoBrigitte — APPROVED** [review]
Kill it

**Pull request merged**